protected/humhub/modules/space/widgets/Menu.php

```diff
@@ -54,6 +54,14 @@ public function init()
             'isActive' => MenuLink::isActiveState('space', 'space', ['index', 'home']),
         ]));
 
+        $this->addEntry(new MenuLink([
+            'label' => Yii::t('SpaceModule.base', 'About'),
+            'url' => $this->space->createUrl('/space/space/about'),
+            'icon' => 'fa-id-card-o',
```

Comment:
Just a small note, rather use icon names without `fa-` here.

Reply:
So a simple `id-card-o` should do? Then doing the same thing for the Stream menu option would also be needed done right?

Reply:
Yes, as noted its not a big deal but the icon abstraction does not require the fa prefix.

Reply:
Should be fixed :smile: